app/src/main/java/org/p2p/wallet/auth/ui/restore/common/CommonRestoreFragment.kt

```diff
@@ -79,6 +79,11 @@ class CommonRestoreFragment :
                 }
                 false
             }
+            toolbar.setOnLongClickListener {
```

Comment:
we do already have DebugSettings opened when buttonSeed is long clicked. check line 98

Reply:
fail, sorry